### DISCUSSION TITLE
laps indication?

### DISCUSSION THREAD

**Discussion by jarss** [OpenTracksApp/OpenTracks#742] [Ideas]
Would be helpful if data could also be shown as "laps" as one of the tabs when done recording. 
currently there's distance, speed, elevation.

 Laps can maybe be indicated if the app catches a GPS location that was visited in the same track already, withing certain distance and  time. So for example, if I pass near a point I already passed within 10 meters, and it's been at least a minute, count it as a "lap". 

This will be similar to what fitbit has on the watch, would also look good on the map when exporting. What do you think?

**Comment by dennisguse** [comment]
Possible yes, but not straight forward; it is actually kind of complicated.
But if somebody finds time why not.
Just create an issue for this.

PS: If you would have time to work on this yourself, I can give you some starting points.
